openupgrade_scripts/apriori.py

```diff
@@ -0,0 +1,29 @@
+""" Encode any known changes to the database here
+to help the matching process
+"""
+
+# Renamed modules is a mapping from old module name to new module name
+renamed_modules = {
+    # odoo
+    # odoo/enterprise
+    # OCA/...
+}
+
+# Merged modules contain a mapping from old module names to other,
+# preexisting module names
+merged_modules = {
+    # odoo
+    # OCA/...
+}
+
+# only used here for upgrade_analysis
+renamed_models = {
+    # odoo
+    # OCA/...
+}
+
+# only used here for upgrade_analysis
+merged_models = {
+    # odoo
+    # OCA/...
+}
```

Comment:
This file is empty. FYI, I began a list of renamed / merged Odoo modules here : 
https://github.com/OCA/OpenUpgrade/issues/4124#issue-1888214543

Reply:
Thanks! Great work on the renamed merge modules. I updated apriori with it and reran the analysis.